app/views/conference/schedule.html.erb

```diff
@@ -25,3 +25,19 @@
       <%= render 'all_events' %>
   <% end %>
 </div>
+
+<%= javascript_tag do %>
```

Comment:
I think we should rather put this onto a different controller method now. Not in a tab. This view is getting complicated!